browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -40,7 +44,7 @@ BraveTabMenuModel::BraveTabMenuModel(
       indices.begin(), indices.end(), [&tab_strip_model](int index) {
         return tab_strip_model->GetWebContentsAt(index)->IsAudioMuted();
       });
-  Build(indices.size());
+  Build(browser, tab_strip_model, indices);
```

Comment:
We have a `CHECK(browser)` as the first line of `BuildItemsForSplitView` which makes me think we should also have a `CHECK(web_contents_)` above instead of the `if` (as when `web_contents_` isn't set, `browser` won't be).

Alternatively, I think it'd be fine if we just didn't call `Build` (or just `BuildItemsForSplitView`).

Reply:
You know, Adding up to your comment, I was like "when browser or web contents could be null when tab context menu is popping up?" So I feel like adding CHECK(browser) and CHECK(web_contents) here in constructor.